tracecat/expressions/functions.py

```
@@ -353,11 +407,142 @@ def to_timestamp_str(x: datetime) -> float:
+    return x.second
+
+
+def extract_minute(x: datetime) -> int:
+    return x.minute
+
+
+def extract_hour(x: datetime) -> int:
+    return x.hour
+
+
+def extract_day_of_week(x: datetime, format: Literal["number", "full", "short"]  = "number") -> int | str:
```

Ruff (0.7.0)

Undefined name `Literal`

(F821)